providers/provider-springmvc/src/test/java/org/apache/servicecomb/provider/springmvc/reference/async/CseAsyncClientHttpRequestTest.java

```diff
@@ -61,8 +61,8 @@ public void teardown() {
 
   @RequestMapping(path = "SpringmvcImpl")
   static class SpringmvcImpl {
```

Comment:
why not just change "SpringmvcImpl" to "CseAsyncClientHttpRequestTestSchema"

Reply:
Agree, change class name would be easier.